pkgs/applications/networking/instant-messengers/franz/generic.nix

```diff
@@ -28,7 +28,7 @@
 # Helper function for building a derivation for Franz and forks.
 
 { pname, name, version, src, meta }:
-stdenv.mkDerivation {
```

Comment:
Can you explain the motivation behind those changes?

Reply:
They were required to make the newest version of Franz work - it would otherwise exit when launched failing to load `libstdc++.so`.